lib/vagrant-openshift/action/try_restart_openshift3.rb

```diff
@@ -26,7 +26,7 @@ def initialize(app, env)
         end
 
         def call(env)
-          sudo(env[:machine], "systemctl try-restart openshift")
+          sudo(env[:machine], "systemctl restart openshift")
```

Comment:
@pweil- I am thinking you want a separate action that starts openshift, installs the router, and stops openshift.  And then add that step to jenkins.  That sound reasonable?